dressvid/src/routes/WatchLaterpage/WatchLaterpage.css

```diff
@@ -0,0 +1,64 @@
+.main-section {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 4rem;
+  margin-left: 18rem;
```

Comment:
Video section is not in center .  we can remove this line .
other than this everything looks good .